icx/src/main.rs

```diff
@@ -405,26 +429,27 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
                 line = input.pop_front().unwrap();
             }
 
-            let transport = ReqwestHttpReplicaV1Transport::create(opts.replica)?;
+            let _transport = ReqwestHttpReplicaV2Transport::create(opts.replica)?;
             match line.as_str() {
-                "read" => {
-                    input.pop_front().unwrap(); // empty line.
-                    line = input.pop_front().unwrap(); // envelope
-                    let envelope = hex::decode(line)?;
-                    let result = transport.read(envelope).await?;
-                    eprint!("Result: ");
-                    println!("{}", hex::encode(result));
-                }
-                "submit" => {
-                    line = input.pop_front().unwrap(); // request id.
-                    let request_id = RequestId::from_str(&line)?;
-                    input.pop_front().unwrap(); // empty line.
-                    line = input.pop_front().unwrap(); // envelope
-                    let envelope = hex::decode(line)?;
-                    transport.submit(envelope, request_id).await?;
-                    eprint!("Request ID: ");
-                    println!("0x{}", hex::encode(request_id.as_slice()));
-                }
+                // "read" => {
```

Comment:
?

Reply:
This code is used, we need to update it as well.

Reply:
I forgot to update this.